src/fraction.ts

```diff
@@ -6,14 +6,23 @@
 // @author zz85
 // @author incompleteopus (modifications)
 
-/* eslint-disable no-underscore-dangle */
-
 import { Vex } from './vex';
+
 export class Fraction {
+  numerator: number;
+
+  denominator: number;
+
+  private static __compareA = new Fraction();
```

Comment:
Can you add a comment here: `Cached objects for comparisons`

Reply:
Ok.
 
Would you change the names to remove the `__`? `fractionCompareA` for example rather than `__compareA`

Reply:
@mscuthbert any proposal on static variable naming convention? This `__` is unique in the program. Would you change to `fractionCompareA`?

Reply:
The `__` contradicts the camel naming convention. I like your suggestion to add `static`, but this one I believe `__` can go away. Would you agree?

Reply:
I was wrong, it does not contradict camelCase. They are ignored at the beginning.
https://eslint.org/docs/rules/camelcase

I will leave them.